phoenix-hive/src/main/java/org/apache/phoenix/hive/constants/PhoenixStorageHandlerConstants.java

```diff
@@ -91,6 +92,7 @@
     public static final String DATE_PATTERN = "'?\\d{4}-\\d{2}-\\d{2}'?";
     public static final String TIMESTAMP_PATTERN = "'?\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2}\\" +
             ".?\\d{0,9}'?";
+    public static final String BIGINT_PATTERN = "[0-9]+L";
```

Comment:
Is the `L` only allowed to be capitalized? Or would `10l` (lowercase 'L') also be valid from the Hive side?